spec/erb_lint/linters/rubocop_spec.rb

```diff
@@ -75,6 +75,59 @@
     it { expect(linter_errors).to eq [arbitrary_error_message(line: 4)] }
   end
 
+  context 'code is aligned to the column matching start of erb tag' do
+    let(:linter_config) do
+      {
+        only: ['Layout/AlignParameters'],
+        AllCops: {
+          TargetRubyVersion: '2.3',
```

Comment:
Can we go full steam on 2.4?

Reply:
I changed it, though this is just a test, not sure it changes much :)